common/src/main/java/io/homeassistant/companion/android/common/data/integration/impl/IntegrationService.kt

```diff
@@ -38,6 +40,12 @@ interface IntegrationService {
         @Header("Authorization") auth: String
     ): Array<DomainResponse>
 
+    @GET("/api/states/{entity}")
+    suspend fun getState(
+        @Header("Authorization") auth: String,
+        @Path("entity") entityId: String
+    ): SingleEntityResponse<Any>
```

Comment:
We don't need to create another object, can just reuse the one we use when requesting all states.  Then if we get a message back we can just throw an exception rather than return null.
```suggestion
    ): EntityResponse<Any>
```